### ISSUE TITLE
Can't import issues from GitHub

### ISSUE THREAD

**Issue by k4y4k** [ralphbean/bugwarrior#742]
Howdy :cowboy_hat_face:!

# tl;dr
`bugwarrior-pull` spits out an error after churning away at my GitHub issues for a bit.

## What I want
`bugwarrior-pull` adds all my issues and PRs from both my private and public repos to my taskwarrior

## What I did

After a bit of muddling around to get this set up, I've run `bugwarrior-pull` and have been greeted by this:
```
INFO:bugwarrior.db:Service-defined UDAs exist: you can optionally use the `bugwarrior-uda` command to export a list of UDAs you can add to your taskr
c file.                                                                                                                                              
INFO:bugwarrior.services:Starting to aggregate remote issues.                                                                                        
INFO:bugwarrior.services:Spawning 1 workers.                                                                                                         
INFO:bugwarrior.services:Working on [github]                                                                                                         
INFO:bugwarrior.services:Done with [github] in 133.078218s                                                                                           
INFO:bugwarrior.services:Done aggregating remote issues.                                                                                             
INFO:bugwarrior.db:Adding 332 tasks                                                                                                                  
INFO:bugwarrior.db:Adding task (bw)Is#23 - Add projects to Work section .. https://github.com/k4y4k/2019-Site/issues/23                              
Traceback (most recent call last):                                                                                                                   
  File "/home/kayak/.local/bin/bugwarrior-pull", line 33, in <module>                                                                                
    sys.exit(load_entry_point('bugwarrior==1.7.0', 'console_scripts', 'bugwarrior-pull')())
  File "/usr/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs) 
  File "/usr/lib/python3.8/site-packages/click/core.py", line 782, in main 
    rv = self.invoke(ctx)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/kayak/.local/lib/python3.8/site-packages/bugwarrior/command.py", line 73, in pull
    synchronize(issue_generator, config, main_section, dry_run)
  File "/home/kayak/.local/lib/python3.8/site-packages/bugwarrior/db.py", line 422, in synchronize
    new_task = tw.task_add(**issue)
  File "/home/kayak/.local/lib/python3.8/site-packages/taskw/warrior.py", line 658, in task_add
    raise KeyError(
KeyError: "Error encountered while creating task;STDOUT: b''; STDERR: b''" 
```

## What I have
Arch Linux, fully updated.

`~/.config/bugwarrior/bugwarriorrc`
```
[general]
  targets = github

[github]
  service = github
  github.default_priority = M
  github.username = k4y4k
  github.login = k4y4k
  github.token =  [a token with repo scope]
```
`task version`
```
task 2.5.1 built for linux
```

## Where I've looked
#444 and #176 look like they're the same thing (444 definitely), but I've reinstalled and restarted `task` and `bugwarrior` a few times and even rebooted my PC. 

What's up?

**Comment by k4y4k** [comment]
LMAO

Yeah I didn't have `taskw` installed, and then, after installing it, everything is A-OK. Maybe a line should be put into the docs somewhere? :eyes:
